docs/reference/msbuild-targets.md

```diff
@@ -409,6 +409,7 @@ Additional restore settings may come from MSBuild properties in the project file
 | NuGetLockFilePath | A custom location for the lock file. The default location is next to the project and is named `packages.lock.json`. |
 | RestoreForceEvaluate | Forces restore to recompute the dependencies and update the lock file without any warning. |
 | RestorePackagesConfig | An opt in switch, that restores projects with packages.config. Support with `MSBuild -t:restore` only. |
+| RestoreUseStaticGraphEvaluation | Tells commandline restore to use static graph MSBuild evaluation instead of the standard evaluation. Static graph evaluation is an experimental feature that's significantly faster for large repos and solutions. |
```

Comment:
Does this also affect Visual Studio? Consider not mentioning command line:

```suggestion
| RestoreUseStaticGraphEvaluation | Opt-in switch to use static graph MSBuild evaluation instead of the standard evaluation. Static graph evaluation is an experimental feature that's significantly faster for large repos and solutions. |
```

If you prefer as is, please split "commandline" into two words :)

Reply:
Let's commit this change because users are very likely to put the MSBuild property into a .csproj & use VS. @nkolev92 any objections?

Reply:
It doesn't affect VS restore at all.

The *entry* project is the one that needs it set. I wanted to avoid those details because it can get confusing.

Reply:
changed it to make it equivalent to `RestorePackagesConfig`.

Reply:
Ah okay. I swore I tried using it in VS and it worked the same. It's been too long to remember though.